lib/p5.js

```diff
@@ -6726,11 +6757,9 @@ amdclean['core_rendering'] = function (require, core_core, core_constants, core_
       document.body.appendChild(c);
     }
     if (r === constants.WEBGL) {
-      if (!this._defaultGraphics) {
-        this._setProperty('_graphics', new p5.Graphics3D(c, this, true));
-        this._defaultGraphics = this._graphics;
-        this._elements.push(this._defaultGraphics);
-      }
+      this._setProperty('_graphics', new p5.Graphics3D(c, this, true));
+      this._defaultGraphics = this._graphics;
```

Comment:
I don't think you added this, but I'm wondering if you remember and can remind me why there is this._graphics and this._defaultGraphics. Is one just the graphics that gets created with the default canvas? Is it necessary to store two pointers to the graphics object or could _defaultGraphics be a boolean flag instead?